pkg/printers/internalversion/describe.go

```diff
@@ -1918,7 +1918,9 @@ func describeService(service *api.Service, endpoints *api.Endpoints, events *api
 		printAnnotationsMultiline(w, "Annotations", service.Annotations)
 		w.Write(LEVEL_0, "Selector:\t%s\n", labels.FormatLabels(service.Spec.Selector))
 		w.Write(LEVEL_0, "Type:\t%s\n", service.Spec.Type)
-		w.Write(LEVEL_0, "IP:\t%s\n", service.Spec.ClusterIP)
+		if service.Spec.ClusterIP != "" {
```

Comment:
Simplest is to have unconditional output.  If you want to conditionalize it, you'd need to relate it to the various service types and deal with exceptional cases.  I recommend leaving it as-is.